src/ontology/mondo-edit.obo

```diff
@@ -228258,6 +228259,7 @@ xref: SCTID:14269005 {source="DOID:3246", source="EFO:0000437"}
 xref: SCTID:404051002 {source="DOID:3246", source="MONDO:equivalentTo", source="EFO:0000437"}
 xref: UMLS:C0206656 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:104910"}
 is_a: MONDO:0005212 {source="DC-OMIM:268210", source="DOID:3246", source="EFO:0000437", source="NCIT:C8971", source="ONCOTREE:ERMS", source="Orphanet:99757"} ! rhabdomyosarcoma
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10964 {source="OMIM:268210"} ! SLC67A1
```

Comment:
This should have been excluded https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv#L7. I will update manually.